arduino/src/arduino.cpp

```diff
@@ -0,0 +1,285 @@
+/**
+ * @file arduino.cpp
+ * @author yarin
+ * @brief  the arduino<-> pi protocol arduino object
+ * @version 0.1
+ * @date 2020-11-21
+ * @copyright Copyright (c) 2020
+ * 
+ */
+
+#include "arduino.h"
+#define RATE 9600
+#define TIMEOUT 10000
+#define READLENGTH 100
+#define SUCCESS 1
+#define DATALEN 1000
+#define COMMENDHEADER "3"
+#define FAILURE -1
+#define NULLCHAR '\0'
```

Comment:
'\0' == NULL == nullptr
and for most cases also == 0

Reply:
fixed